packages/web-components/src/dialog/dialog.options.ts

```diff
@@ -11,3 +12,19 @@ export const DialogType = {
 } as const;
 
 export type DialogType = ValuesOf<typeof DialogType>;
+
+/**
+ * Predicate function that determines if the element should be considered a dialog.
+ *
+ * @param element - The element to check.
+ * @param tagName - The tag name to check.
+ * @returns true if the element is a dialog.
+ * @public
+ */
+export function isDialog(element?: Node | null, tagName: string = '-dialog'): element is Dialog {
```

Comment:
Is there any case where the dialog tag name doesn't end with `-dialog`? I'm just wondering if the `tagName` parameter is necessary.

Reply:
Similar pattern we follow else where... Just an optional way of updting the filter if you register a new custom dialog element that ends with something different